sync.sh

```diff
@@ -1,22 +1,60 @@
-#!/bin/sh -ex
+#!/bin/bash -e
 
 REPO=../auto-opam-repository
 UPSTREAM=git://github.com/ocaml/opam-repository
 GEN=`pwd`/generate.native
 PULL=`pwd`/create_pull_request.native
-V=4.03.0
 if [ ! -d ${REPO} ]; then
   git clone git@github.com:bactrian/opam-repository ${REPO}
 fi
 
 BRANCH=sync-prs-`date +%s`
 HRDATE=`date +%c`
 cd $REPO
+# @@DRA Cleaner to ensure that origin/upstream is same as $UPSTREAM
+#       and do a fetch followed by hard reset to upstream
+git checkout -f master
+git reset --hard
 git pull $UPSTREAM master
 git checkout -b $BRANCH
 
-rm -rf compilers/${V}/${V}+pr*
-$GEN -compiler-version ${V}
+# This becomes a little less scary-looking in OPAM 2.0 when it's packages/ocaml/*+pr[0-9]*
+rm -rf compilers/*/*+pr[0-9]*
```

Comment:
Must use a better regexp, or we catch 4.01.0+profile (not that that switch is actually needed any more)